approval_polls/views.py

```diff
@@ -79,7 +79,7 @@ def add(request):
     if not len(choices):
       return render(request, 'approval_polls/error.html', {'error_message': 'At least one choice is required'})
 
-    p = Poll(question=question, pub_date=timezone.now())
+    p = Poll(question=question, pub_date=timezone.now(), username=request.POST['username'].strip())
```

Comment:
@Brianjr0428 not sure why you want to use to_field though, going with the default primary key is the preferred option unless there is a specific reason not to.